tests/baselines/reference/tsConfig/Initialized TSConfig with incorrect compiler option/tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "compilerOptions": {
     /* Basic Options */
     "target": "es5",                          /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', or 'ESNEXT'. */
-    "module": "commonjs",                     /* Specify module code generation: 'none', commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */
```

Comment:
not sure why yo had to change these.

Reply:
These fixes are made by `gulp baseline-accept`.

Completely unrelated to the issue, but I suppose it would not hurt to fix a typo in the comments.

Should I revert it?

Reply:
looks like we did not catch these failures. that should be fixed by https://github.com/Microsoft/TypeScript/pull/18583

Reply:
Great! I've removed these changes from the PR.